### PR TITLE
Improve regional count accuracy for CEs

### PR DESCRIPTION
**Pull request by NickGrantX**
# Motivation and Context
The regional counts are drifting further and further from the 'real' number, because we are not counting CEs correctly. We should make this useful stat more accurate.

# What has changed
Used response events instead of case marker to count responses.

# How to test?
Run it. Does it spit out meaningful numbers?

# Links
Trello: https://trello.com/c/1oosPUDi

### CODE REVIEW

**Comment by NickGrantX** [comment]
[Improve regional counts metric accuracy](https://trello.com/c/1oosPUDi/2093-improve-regional-counts-metric-accuracy)

**Review by hbrace — APPROVED** [review]

File: monitoring/regional_counts.py [hunk 1]
```diff
@@ -10,11 +10,10 @@
...
            ELSE 'Unknown Region: ' || counts.region END,
        counts.region_count
-FROM (SELECT LEFT(region, 1) AS region, COUNT(*) AS region_count
-      FROM casev2.cases
-      WHERE receipt_received = true
-        AND action_plan_id = 'c4415287-0e37-447b-9c3d-1a011c9fa3db'
-      GROUP BY LEFT(region, 1)) AS counts
+FROM (SELECT LEFT(c.region,1) as region, COUNT(*) AS region_count
+      FROM casev2.uac_qid_link q, casev2.cases c
+      WHERE q.active='f' AND q.caze_case_id = c.case_id AND c.action_plan_id = 'c4415287-0e37-447b-9c3d-1a011c9fa3db'
```
Comment (AdamHawtin) [inline monitoring/regional_counts.py]: The active flag being false doesn't equate to a response being received, they can be deactivated for other reasons.

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
The active flag being false doesn't necessarily mean a response has been received, it can be deactivated for other reasons.

File: monitoring/regional_counts.py (hunk 1 above)
Comment (NickGrantX) [reply monitoring/regional_counts.py]: good point... I was going to count via events, but forgot

Comment (NickGrantX) [reply monitoring/regional_counts.py]: Fixed

**Review by LukeLoze1 — APPROVED** [review]

File: monitoring/regional_counts.py [hunk 2]
```diff
@@ -2,23 +2,51 @@
...
     print(json.dumps(regional_counts))
+
+
+def get_response_events():
+    sql_query = """
+    SELECT CASE counts.region
+               WHEN 'E' THEN 'England Responses'
+               WHEN 'W' THEN 'Wales Responses'
+               WHEN 'N' THEN 'Northern Ireland Responses'
+               ELSE 'Unknown Region: ' || counts.region END,
```
Comment (AdamHawtin) [inline monitoring/regional_counts.py]: I think the `Unkown Region` count would be indistinguishable from the receipted count above in the log metrics (not that we should actually see any unknown regions).

**Review by AdamHawtin — APPROVED** [review]
👍

**Pull request merged**